petab/C.py

```diff
@@ -126,7 +126,9 @@
 
```

Comment:
just defenitions of new global parameters ORDER and SCATTER. So that in future if we agree that in visu file "order" should be replaced by "equidistant", codewise it only has to be done in this file.